src/CMakeLists.txt

```diff
@@ -359,6 +359,7 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Emscripten")
       rigel_core
     )
     target_link_options(${main_exe_name} PRIVATE
```

Comment:
`target_link_options` was introduced in CMake 3.13. You should update the version in the call to `cmake_minimum_required` in the top-level `CMakeLists.txt` file.

Reply:
Good catch, thanks!